### ISSUE TITLE
Create new collection to capture AST approval stage

### ISSUE THREAD

**Issue by vpaba** [nvs-vocabs/ArgoVocabs#33]
A new NVS collection needs to be created to capture the AST approval stages, and will include only two concepts:
- PILOT
- APPROVED (or ACCEPTED, or GLOBAL IMPLEMENTATION - to be decided!)

This solution has been agreed by the @nvs-vocabs/avtt in March 2022 as the most efficient way to clearly identify which sensors (in R27), parameters (in R03) etc. belong to which approval stage, by establishing NVS mappings to the corresponding term in the new AST approval stage collection. These mappings can be reviewed if a sensor goes from pilot to approved.

**Comment by vturpin** [comment]
Not sure this question goes here. 
What is the process to suggest an evolution of the Argo format ? 

I am asking as we are almost reaching an agreement amongst the participant of different issues : #2 and #5 . 

Before we start working toward implementation at OceanOPS and NVS, we would like to understand the procedure to have those issues endorse by Argo during the next ADMT ?
My understanding of the process so far is : 
1) raise an issue and discuss it on GitHub
2) reach a first agreement amongst contributors to the issue
3) draft a proposal for discussion and agreement
4) submit proposal to ADMT for official endorsement
5) adoption in the argo format
6) Implementation by DAC

Does this process make sense ?

**Comment by tcarval** [comment]
In table R27 "sensor model", we have the column "parameter status":
- operational
- pilot
- obsolete

**Comment by vpaba** [comment]
This issue was dropped at ADMT-23, as action no longer needed. Please re-open in case!
